microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md

```diff
@@ -0,0 +1,209 @@
+## Use your source maps to unminify performance profiles
+
+<!-- Title: The Performance tool can now display unminified call stacks -->
+<!-- Subtitle: Use the new Unminify button in the Performance tool to download an unminified version of the performance profile you recorded. -->
+
+In previous versions of Microsoft Edge, recording a performance profile in the Performance tool would produce a minified call stack. The call stack was not using source maps, even if you hosted them on your server, to unminify the call stack.
+
+Starting in Microsoft Edge version 99, you can unminify a call stack in a performance profile, as follows:
+1. After recording a profile, in the **Performance** tool, click the new **Unminify** button.
+1. Download the unminified performance profile.<!--how?-->
+1. Import the unminified performance profile into the **Performance** tool.<!--how?-->
+
+The **Unminify** button will use source maps, provided that they are hosted side-by-side with your production code, to unminify the call stack in the **Performance** tool.
+
+The minified call stack in the **Performance** tool shows function names as **b**, **O**, **Xt**, and **bn**:
+
+![The minified call stack in the Performance tool.](../../media/2022/03/minified-call-stack-performance-tool.png)
+<!-- Image 1: Minified call stack in Performance tool
+Navigate to: https://outlook-sdf.office.com/mail/
+Open DevTools > Performance
+Record a performance profile while scrolling through your inbox
+Select a small section of the profile and look at the call stack       1380ms-1400ms
+Draw red highlights around function names like n, O, i, Xt, Bn, Yn. It won't be these exactly in your screenshot but you can see these are minified function names
+Remember the section of the profile that you are looking at
+-->
+
+The unminified call stack in the **Performance** tool now shows those functions labeled with meaningful names: **invokeFunc (b)**, **executeAction (O)**, **endBatch (Xt)**, and **runReactions (bn)**:
+
+![The unminified call stack in the Performance tool.](../../media/2022/03/unminified-call-stack-performance-tool.png)
+<!-- Image 2: Unminified call stack
+Select the Unminify button
+Save the unminified profile (it'll be in the .json file format) to your Downloads folder
+Select Load profile and load the unminified perf profile you just downloaded
+Select the same section of the profile you looked at in the first image
+Draw red highlights around function names that should now be unminified. In my demo, 
+Xt => endBatch
+bn => runReactions
+i => finalDispatch
+ -->
+
+See also:
+
+* [Unminifying function names in DevTools performance profiles](https://blogs.windows.com/msedgedev/2022/02/03/unminifying-function-names-in-devtools-performance-profiles/) at the Microsoft Edge Blog.
+* [Unminify performance profiles (temp. PR link)](https://review.docs.microsoft.com/microsoft-edge/devtools-guide-chromium/evaluate-performance/unminify?branch=pr-en-us-1784).
```

Comment:
I would prefer if we can land [this PR](https://github.com/MicrosoftDocs/edge-developer/pull/1784) and then link to it from here but if we can't make it work in time, we can always add the link later and just have the link to the blog for now

Reply:
I did a rewrite, Writer/Editor pass, on that PR, have to fix a bold tag, wish to have Patrick's input on differentiation of benefits of source maps vs deminification vs ADO secure source maps server, in top of that article.  Near-ready to merge that PR.